addons/resource.language.en_gb/resources/strings.po

```diff
@@ -10976,11 +10976,11 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#20369"
-msgid "Show plot for unwatched items"
+msgid "Show plot for unwatched items (Hides plot and thumb when disabled)"
 msgstr ""
 
 msgctxt "#20370"
-msgid "* Hidden to prevent spoilers *"
+msgid "* Plot and thumbnail hidden to prevent spoilers *"
```

Comment:
Because there could be cases where the viewtype doesnt contain a thumb at all. the label would be misleading then.